packages/manager/apps/web/client/app/domain/domains.routing.js

```diff
@@ -17,7 +17,9 @@ export default /* @ngInject */ ($stateProvider) => {
         .injector()
         .getAsync('resources')
         .then((resources) =>
-          resources.data.length === 0 && !transition.params().filter
+          resources.data.length === 0 &&
+          transition.params().filter &&
```

Comment:
Is `filter` param always set ?

Reply:
normally it should as it comes from layout helper in which we set:
```javascript
stateParams = {
  ...,
  filter: {
    value: '[]',
    squash: true
  }
}
```